### ISSUE TITLE
FileListing could not check dir with sticky bit shown as 'T' in ls command

### ISSUE THREAD

**Issue by TripleJQK** [RedHatInsights/insights-core#1121]
In insights/core/__init__.py:
~~~
1010 class FileListing(Parser):
1011     """
1012     Reads a series of concatenated directory listings and turns them into
1013     a dictionary of entities by name.  Stores all the information for
1014     each directory entry for every entry that can be parsed, containing:
1015 
:
1079     # I know I'm missing some types in the 'type' subexpression...
1080     # Modify the last '\S+' to '\S*' to match where there is no '.' at the end
1081     perms_regex = r'^(?P<type>[bcdlps-])' +\
1082         r'(?P<perms>[r-][w-][sSx-][r-][w-][sSx-][r-][w-][xsSt-]([.+-])?)'
:
~~~

it seems that `perms_regex` can not Match a 1774 dir, which the sticky bit show as `T`:
~~~
---------------'T'-------------------
# ls -ld ls
drwxrwxr-T. 2 root root 40 Apr 20 14:48 ls
# stat ls
  File: ls
  Size: 40        	Blocks: 0          IO Block: 4096   directory
Device: 2ah/42d	Inode: 10817573    Links: 2
Access: (1774/drwxrwxr-T)  Uid: (    0/    root)   Gid: (    0/    root)
Context: unconfined_u:object_r:user_tmp_t:s0
Access: 2018-04-20 14:48:44.500872552 +0800
Modify: 2018-04-20 14:48:44.500872552 +0800
Change: 2018-04-20 14:48:58.180456351 +0800
 Birth: -

------------'t'-----------------
# ls -ld ls
drwxrwxrwt. 2 root root 40 Apr 20 14:48 ls
# stat ls
  File: ls
  Size: 40        	Blocks: 0          IO Block: 4096   directory
Device: 2ah/42d	Inode: 10817573    Links: 2
Access: (1777/drwxrwxrwt)  Uid: (    0/    root)   Gid: (    0/    root)
Context: unconfined_u:object_r:user_tmp_t:s0
Access: 2018-04-20 14:48:44.500872552 +0800
Modify: 2018-04-20 14:48:44.500872552 +0800
Change: 2018-04-20 14:57:56.050091646 +0800
 Birth: -
~~~

**Comment by miclark** [comment]
@TripleJQK, the ls parser has been reworked since this was opened.  Is this still an issue for you?

**Comment by koalakangaroo** [comment]
@xiangce - can you check if it is fixed or not?

**Comment by xiangce** [comment]
I'll have a look to it.

**Comment by xiangce** [comment]
Confirmed. It's already fixed by #1360
